src/themes/default/resource/i18n/en-US.csv

```diff
@@ -114,6 +114,7 @@
 "My loyalty card","My loyalty card"
 "My product reviews","My product reviews"
 "Wishlist","Wishlist"
+"Clear wishlist","Clear wishlist"
```

Comment:
do we have `Are you sure you would like to remove all the items from the wishlist?` text here as well?

Reply:
Now we do ;)